test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -0,0 +1,121 @@
+Import-Module $PSScriptRoot\..\..\Common\Test.Helpers.psm1
+
+Describe "Test-ModuleManifest tests" -tags "CI" {
+
+    AfterEach {
+        Remove-Item -Recurse -Force -ErrorAction SilentlyContinue testdrive:/module
+    }
+
+    It "module manifest containing paths with backslashes or forwardslashes are resolved correctly" {
+
+        New-Item -ItemType Directory -Path testdrive:/module
+        New-Item -ItemType Directory -Path testdrive:/module/foo
+        New-Item -ItemType Directory -Path testdrive:/module/bar
+        New-Item -ItemType File -Path testdrive:/module/foo/bar.psm1
+        New-Item -ItemType File -Path testdrive:/module/bar/foo.psm1
+        $testModulePath = "testdrive:/module/test.psd1"
+        $fileList = "foo\bar.psm1","bar/foo.psm1"
+
+        New-ModuleManifest -NestedModules $fileList -RootModule foo\bar.psm1 -RequiredAssemblies $fileList -Path $testModulePath -TypesToProcess $fileList -FormatsToProcess $fileList -ScriptsToProcess $fileList -FileList $fileList -ModuleList $fileList
+
+        Test-Path $testModulePath | Should Be $true
+
+        # use -ErrorAction Stop to cause test to fail if Test-ModuleManifest writes to error stream
+        Test-ModuleManifest -Path $testModulePath -ErrorAction Stop | Should BeOfType System.Management.Automation.PSModuleInfo
+    }
+
+    It "module manifest containing missing files returns error" -TestCases `
```

Comment:
Typical use of `-TestCases` is to introduce a variable - that is probably easier to read.

If there is a good reason to inline the argument - you should prefer parens instead of a backtick.
Extra indentation on the test cases might also make it more readable, so maybe format like:

```PowerShell
It "should work" -TestCases (
        @{rootModuleValue = "bar.psm1"},
        @{rootModuleValue = "bar.dll"}
) {
    param($rootModuleValue)
}
```